docs/source/notebooks/dap_subset.ipynb

```diff
@@ -497,20 +497,21 @@
        "    NCO:            4.0.9"
       ]
      },
-     "execution_count": 12,
+     "execution_count": 2,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
```

Comment:
In the nbval doc, there is a space between `#` and `NBVAL_IGNORE_OUTPUT`.  Not sure if it has to be exactly like that or not.  If you run `make test-notebooks`, everything pass, even without lax?  If yes then all good.

Reply:
Yes.